test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
```

Comment:
I know.
BTW, it also has `Safari` and (of course) `Edge` (for interoperability reasons - or so I've read :P).